### PR TITLE
Fix crash when no bluetooth available

### PR DESCRIPTION
**Pull request by vjr**
Followup fixes to #132 and https://github.com/elementary/switchboard/issues/180

### CODE REVIEW

**Comment by vjr** [comment]
With this fix, the plug UI still opens and the bluetooth toggle still "works" with the message "no bluetooth devices detected" and a timeout message printed on the console, so not sure if this needs further UI updates.

File: src/Services/Manager.vala [hunk 1]
```diff
@@ -69,6 +69,7 @@ public class Bluetooth.Services.ObjectManager : Object {
                 object_manager_proxy_get_type,
                 null
             );
+            if (object_manager == null) return;
```
Comment (jeremypw) [inline src/Services/Manager.vala:72]: Contravenes elementary code style: Should be 
```
if (object_manager == null) {
    return;
}
```

File: src/Services/Manager.vala [hunk 2]
```diff
@@ -233,6 +238,7 @@ public class Bluetooth.Services.ObjectManager : Object {
     }
 
     private async void create_agent (Gtk.Window? window) {
+        if (object_manager == null) return;
```
Comment (jeremypw) [inline src/Services/Manager.vala:241]: Code style - see above comment

**Review by jeremypw — CHANGES_REQUESTED** [review]
Couple of minor code style issues.  This should work (havent tested it yet) to prevent a crash.  It would be nicer to have a more accurate message displayed but that can be left for a future PR if desired (raise a new issue).  The main thing is to stop the crash.

File: src/Services/Manager.vala (hunk 2 above)
Comment (vjr) [reply src/Services/Manager.vala]: Addressed!

File: src/Services/Manager.vala (hunk 1 above)
Comment (vjr) [reply src/Services/Manager.vala]: done!

**Review by jeremypw — APPROVED** [review]
Can confirm it fixes the issue. Thanks!

**Pull request merged**